tests/unit/bokeh/test_settings.py

```diff
@@ -90,11 +90,12 @@ def test_types(self) -> None:
         assert bs.settings.minified.convert_type == "Bool"
         assert bs.settings.perform_document_validation.convert_type == "Bool"
         assert bs.settings.simple_ids.convert_type == "Bool"
-        assert bs.settings.strict.convert_type == "Bool"
         assert bs.settings.xsrf_cookies.convert_type == "Bool"
 
         assert bs.settings.py_log_level.convert_type == "Log Level"
 
+        assert bs.settings.validation_level.convert_type == "Validation Level String"
```

Comment:
Let's make this "Validation Level", to be consistent with "Log Level"

```suggestion
        assert bs.settings.validation_level.convert_type == "Validation Level"
```